README.md

```diff
@@ -6,7 +6,7 @@
 [![Snap](https://snapcraft.io/git-machete/badge.svg)](https://snapcraft.io/git-machete)
 [![Snap build status](https://build.snapcraft.io/badge/VirtusLab/git-machete.svg)](https://build.snapcraft.io/user/VirtusLab/git-machete)
 
-![](https://raw.githubusercontent.com/VirtusLab/git-machete/master/logo.png)
+<img src="https://raw.githubusercontent.com/VirtusLab/git-machete/master/logo.png" width="180" height="180" align="left" />
```

Comment:
Looking at the https://github.com/isocpp/CppCoreGuidelines as a reference...

It would be nice to have a full-width-logo for our readmes/docs. Centering is rather a temporary solution.